docs/data/joy/components/menu/MenuToolbarExample.js

```diff
@@ -148,11 +148,8 @@ export default function MenuToolbarExample() {
           }}
           menu={
             <Menu
-              onClose={(event) => {
+              onClose={() => {
                 menus.current[0]?.focus();
-                if (event && event.relatedTarget !== menus.current[0]) {
```

Comment:
`event` argument was removed from `onClose` [in recent changes](https://github.com/mui/material-ui/blob/master/packages/mui-joy/src/Menu/MenuProps.ts#L56). so there is no point running this logic